xarray/coding/cftime_offsets.py

```diff
@@ -237,11 +237,7 @@ def _shift_month(date, months, day_option="start"):
         day = _days_in_month(reference)
     else:
         raise ValueError(day_option)
-    # dayofwk=-1 is required to update the dayofwk and dayofyr attributes of
-    # the returned date object in versions of cftime between 1.0.2 and
-    # 1.0.3.4.  It can be removed for versions of cftime greater than
-    # 1.0.3.4.
-    return date.replace(year=year, month=month, day=day, dayofwk=-1)
+    return date.replace(year=year, month=month, day=day)
```

Comment:
```suggestion
    if LooseVersion(cftime.__version__) < LooseVersion("1.0.4"):
        # dayofwk=-1 is required to update the dayofwk and dayofyr attributes of
        # the returned date object in versions of cftime between 1.0.2 and
        # 1.0.3.4.  It can be removed for versions of cftime greater than
        # 1.0.3.4.
        return date.replace(year=year, month=month, day=day, dayofwk=-1)
    else:
        return date.replace(year=year, month=month, day=day)
```
I think you may need to add `from distutils.version import LooseVersion` to the top of the module here too.